docs/sources/operations/storage/wal.md

```diff
@@ -95,6 +95,18 @@ Statefulsets are significantly more cumbersome to work with/upgrade/etc. Much of
 
 In this case, try `kubectl -n <namespace> delete sts ingester --cascade=false`. This will leave the pods alive but delete the statefulset. Then you may recreate the (updated) statefulset and one-by-one start deleting the `ingester-0` through `ingester-n` pods _in that order_, allowing the statefulset to spin up new pods to replace them.
 
+#### Scaling Down Using `/flush_shutdown` Endpoint and Lifecycle Hook
+
+1. **StatefulSets for Ordered Scaling Down**: Loki's ingesters should be scaled down one by one, which is efficiently handled by Kubernetes StatefulSets. This ensures an ordered and reliable scaling process, as described in the [Deployment and Scaling Guarantees](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#deployment-and-scaling-guarantees) documentation.
```

Comment:
You describe the behavior of `OrderedReady` here, but the statefulset is configured for `Parallel`.

https://github.com/grafana/loki/blob/425736614c43f3435e52bc1e5a2295dfcbb6418a/production/helm/loki/templates/write/statefulset-write.yaml#L15

Reply:
Updated, thanos good catch